Gemfile

```diff
@@ -16,7 +16,11 @@ else
   gem 'gds-api-adapters', '25.1.0'
 end
 
-gem 'govuk_message_queue_consumer', '~> 2.1.0'
+if ENV["MESSAGE_QUEUE_CONSUMER_DEV"]
```

Comment:
Could we skip this? I know this pattern is used a lot, but it doesn't save too much typing and makes the Gemfile less readable.